tigerpath/majors_and_certificates/majors/CEE.json

```diff
@@ -383,7 +383,7 @@
           "name": "Engineering design",
           "max_counted": 1,
           "min_needed": 2,
-          "explanation": "Engineering design (two courses):\n\nCEE 471 Introduction to Water Pollution Technology\nCEE 477 Engineering Design for Sustainable Development",
+          "explanation": "Engineering design (two courses):\nCEE 471: Introduction to Water Pollution Technology,\nCEE 477: Engineering Design for Sustainable Development",
```

Comment:
You can probably delete the courses in the explanation here.

Reply:
Actually never mind, you can probably leave it. There seems to be a lot of these explanations.